### ISSUE TITLE
Adding Momentum, weight decay and learning rate schedule didn't improve the results

### ISSUE THREAD

**Issue by gaopeng-eugene** [DrSleep/tensorflow-deeplab-lfov#10]
Hi, I follow the resnet branch by adding momentum, weight decay and learning rate schedule. However, the results didn't improve much. The final model are worse than the model.ckpt-pretrained.

**Comment by DrSleep** [comment]
I cannot really help with this one as I have not been doing these experiments.
